sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -78,29 +78,30 @@ Install the Azure Text Analytics client library for Python with [pip][pip]:
 pip install azure-ai-textanalytics --pre
```

Comment:
I can't comment on it, so line 58:
`Interactions with this service using this client library begins with an ......` something like this?